### PR TITLE
Redirect legacy Reporting API article to web.dev Reporting API article

### PR DESCRIPTION
**Pull request by maudnals**
- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Review by agektmr — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
I approved it, but you may want to fix the indent.

File: src/content/en/_redirects.yaml [hunk 1]
```diff
@@ -189,6 +189,9 @@ redirects:
 - from: /web/updates/2018/07/reportingobserver
   to: https://web.dev/reporting-observer/
 
+  - from: /web/updates/2018/09/reportingapi
```
Comment (agektmr) [inline src/content/en/_redirects.yaml:192]:
```suggestion
- from: /web/updates/2018/09/reportingapi
```

File: src/content/en/_redirects.yaml [hunk 2]
```diff
@@ -189,6 +189,9 @@ redirects:
...
+    to: https://web.dev/reporting-api/
```
Comment (agektmr) [inline src/content/en/_redirects.yaml:193]:
```suggestion
  to: https://web.dev/reporting-api/
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by maudnals** [comment]
Thanks Eiji
This is ready to merge!

**Review by agektmr — APPROVED** [review]
LGTM

**Pull request merged**